### ISSUE TITLE
Safe shutdown in case of power loss to prevent SD corruption 

### ISSUE THREAD

**Issue by mgrouch** [m5stack/CoreMP135_buildroot-external-st#7]
It’s not clear how safe shutdown in case of power loss to prevent SD corruption can be done with Debian image and coremp135. 

Could it be added to the documentation?

thanks

**Comment by dianjixz** [comment]
Just execute `poweroff` or perform a `sync` operation before power loss. So, have you encountered the issue of file system corruption?

**Comment by mgrouch** [comment]
I haven’t experienced SD card corruption. 
I did some reading and I think I can close this issue: 

Industrial or high endurance SD card has circuits protecting it from power loss. 
The file system is ext4 which is journaling file system. So on reboot it will restore file system integrity. Journaling file system guarantees file system integrity but not data integrity. Data integrity protections are functions of each individual application. 

So bottom line you just need good SD card. None of them designed to last forever. So have a spare too.
